src/components/EditImageModal/index.jsx

```diff
@@ -234,13 +269,41 @@ export default class EditImageModal extends React.Component {
   }
 
   onStatusAlertClose = () => {
-    this.imageSourceInputRef.focus();
+    if (this.state.pageNumber === 1) {
+      this.imageSourceInputRef.focus();
+    } else if (this.state.pageNumber === 2) {
+      this.previousPageButtonRef.focus();
```

Comment:
I think we should add a case where `this.state.shouldShowPreviousButton` is false (i.e. editing an existing image), since this will throw the same error as in the componentDidUpdate method. I'm not sure what we should focus on. The first fieldset?

Reply:
Good catch -- I think the first Fieldset sounds good, may end up being the search bar instead once that PR merges in.